functions/__fzf_display_value_or_error.fish

```diff
@@ -1,10 +1,12 @@
 # helper function for __fzf_search_shell_variables
-function __fzf_display_value_or_error --argument-names variable_name --description "Displays either the value of the variable passed in, or an informative message if it is not available."
-    if set --query $variable_name
-        echo $$variable_name
+function __fzf_display_value_or_error --argument-names variable_name variable_file --description "Displays either the value of the variable passed in, or an informative message if it is not available."
+    set matches (cat $variable_file | string match --regex "^\\\$$variable_name(?::|\[).+")
+
+    if test -n "$matches"
+        string replace --regex "^\\\$$variable_name(?:: (.+)|(\[.+\]): \|(.+)\|)" '\$1\$2 \$3' $matches
     else
         set_color red
-        echo "$variable_name was not exported to this process so its value cannot be displayed." >&2
+        echo "$variable_name was not dumped so its value cannot be displayed." >&2
```

Comment:
The error message is an abstraction leak. The user doesn't know that we use `set --show | psub` so they will be confused by why the error message is saying it wasn't dumped.

Reply:
We does *export the variable into* a temporary file, but not in the meaning of *export a variable to a subprocess*. I was thinking that keeping the old message may mislead the user. But I guess it's probably no difference.